### PR TITLE
🏴󠁧󠁢󠁷󠁬󠁳󠁿 Payments widgets changes

### PR DESCRIPTION
**Pull request by WRadoslaw**
#3700

### CODE REVIEW

**Comment by attemka** [comment]
seems that the `PayoutsWelcomeDialogContent` should be changed according to the [figma designs](https://www.figma.com/file/ZFRXsyGvnBXnkDUUGc5vyR/My-payments?node-id=321%3A203&t=IH98CI57oMlkypDC-0)
image

File: packages/atlas/src/components/Table/Table.styles.ts [hunk 1]
```diff
@@ -47,3 +47,22 @@ export const Td = styled(Text)`
...
+
+  p {
+    text-align: center;
+    margin-top: ${sizes(2)};
+  }
+
+  h5 {
+    margin-top: ${sizes(10)};
+    text-align: center;
+  }
```
Comment (drillprop) [inline packages/atlas/src/components/Table/Table.styles.ts]: I'm probably nitpicking here, but I would avoid targeting tags in CSS whenever we can. Let's just create: 
```tsx
const EmptyTableTitle = styled(Text)`
// styles
`
const EmptyTableDescription = styled(Text)`
// styles
`
```

Comment (attemka) [reply packages/atlas/src/components/Table/Table.styles.ts]: @drillprop agree, would be easier to navigate in component by reading the styleds names

**Review by attemka — APPROVED** [review]

File: packages/atlas/src/components/Table/Table.styles.ts [hunk 2]
```diff
@@ -47,3 +47,24 @@ export const Td = styled(Text)`
...
+  justify-items: center;
+  margin: ${sizes(10)} auto;
+  padding: 0 ${sizes(4)};
+`
+
+export const EmptyTableHeader = styled(Text)`
+  h5 {
+    margin-top: ${sizes(10)};
+    text-align: center;
+  }
```
Comment (drillprop) [inline packages/atlas/src/components/Table/Table.styles.ts]:
```suggestion
  margin-top: ${sizes(10)};
  text-align: center;
```

**Review by drillprop — CHANGES_REQUESTED** [review]

**Review by drillprop — APPROVED** [review]

**Pull request merged**